cmd/monaco/generate/schemas/schemagenerator.go

```diff
@@ -0,0 +1,104 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package schemas
+
+import (
+	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/json"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/mutlierror"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/topologysort"
+	configErrors "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/config/errors"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/config/parameter"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/config/parameter/reference"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/converter"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/delete"
+	deploy "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/deploy/errors"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/graph"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest"
+	manifestLoader "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest/loader"
+	manifestWriter "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest/writer"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/persistence/account"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/persistence/config"
+	project "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/project/v2"
+	sortErrors "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/project/v2/sort/errors"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/rest"
+	"github.com/spf13/afero"
+	"path/filepath"
+)
+
+var errorStructs = []interface{}{
+	json.JsonValidationError{},
```

Comment:
note: this sucks, but I found no decent way to find all errors automatically so for now they're manually listed.
From an enduser perspective, the schemas for manifest, config and account resources are much more valuable anyway so I didn't spend much time on errors

Maybe we could with reflections look for everything that implements the Error interface? 🤔

Reply:
Removed the generation of errors schemas as discussed (removed as refactoring commit)